megamek/src/megamek/common/AmmoType.java

```diff
@@ -1373,7 +1373,7 @@ public static void initializeTypes() {
                 .setPrototypeFactions(F_TH)
                 .setProductionFactions(F_TH)
                 .setReintroductionFactions(F_FS,F_LC)
-                .setStaticTechLevel(SimpleTechLevel.STANDARD),"373,TO"));
```

Comment:
How do we handle things on two pages? This is on pg 373-374 of TO

Reply:
We’ve always just used one page and tried to be the first page.

Reply:
As far as I can see this is never referenced, only saved. So I'd just give the first page.